tests/cupyx_tests/scipy_tests/ndimage_tests/test_filters.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+
+import numpy
+
+from cupy import testing
+import cupyx.scipy.ndimage  # NOQA
+
+try:
+    import scipy.ndimage  # NOQA
+except ImportError:
+    pass
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(3, 4), (2, 3, 4), (1, 2, 3, 4)],
+    'ksize': [3, 4],
+    'mode': ['reflect', 'constant', 'nearest', 'mirror', 'wrap'],
+    'cval': [0.0],
+    'origin': [-1, 0, 1],
+    'output': [None, numpy.float32, numpy.float64],
```

Comment:
If mixed dtype input/weights are to be supported as in SciPy, please add some test cases for that. 
i.e.  floating input, integer weights
       integer input, floating weights

Reply:
Thanks. I added those test cases for mixed dtype input/weights to `TestConvoluteAndCorrelate`.